cloudmarker/events/azlogprofilemissingactivitytypeevent.py

```diff
@@ -0,0 +1,120 @@
+"""Microsoft Azure Log Profile Missing Activity Type Event.
+
+This module defines the :class:`AzLogProfileMissingActivityTypeEvent` class
+that identifies if a log profile which is not enable for all the activity
+types i.e. Write, Delete and Action. This plugin works on the log profile
+properties found in the ``raw`` bucket of ``log_profile`` records.
+"""
+
+
+import logging
+
+from cloudmarker import util
+
+_log = logging.getLogger(__name__)
+
+
+class AzLogProfileMissingActivityTypeEvent:
```

Comment:
I would like this class name to follow the Azure terminology of "category". The key name in the JSON response is `categories`. The CIS benchmark document for Azure also refers to it as "category". My suggestion: `AzLogProfileMissingCategoryEvent`.

The record type would then become: `log_profile_missing_category_event`. I think these names are a little bit neater.

Reply:
Renamed to `AzLogProfileMissingCategoryEvent ` also event/record type changed to `log_profile_missing_category_event `